ubuntu.sh

```diff
@@ -540,8 +549,26 @@ nanoagent_json() {
   "api_port":"8570",
   "route_http_port":"80",
   "route_tls_port":"443",
-  "data_file":"/var/db/nanoagent/bolt.db"
+  "data_file":"/var/db/nanoagent/bolt.db",
+  "on_container_start": "/opt/nanoagent/hooks/container_on_start.sh",
+  "on_container_stop": "/opt/nanoagent/hooks/container_on_stop.sh"
+}
+END
+}
+
+nanoagent_container_start() {
+  cat <<END
```

Comment:
@notxarb, does the `$(docker inspect...)` need escaped too. If you wrap this in quotes, you wouldn't need to escape the variables:
```
cat <<'END'
docker exec $1
END
```

Reply:
That one should be escaped, too, yes. We can't quote the entire script, though, because `$MTU` needs to be interpolated.